manim/mobject/geometry.py

```diff
@@ -912,6 +912,29 @@ def __init__(self, *args, **kwargs):
 
 
 class CubicBezier(VMobject):
+    """
+    Example
+    -------
+
+    .. manim:: BezierSplineExample
+        :save_last_frame:
+
+        class BezierSplineExample(Scene):
+            def construct(self):
+                np.random.seed(42)
```

Comment:
There is no randomness in this example, is there?

```suggestion
```

Reply:
yep, that was only used in a previous version of this script